tests/test_timestamp.py

```diff
@@ -455,6 +455,7 @@ def test_to_datetime(self):
             (datetime(1970, 1, 1, 0, 0, 0, 0, tz.gettz('UTC')), Timestamp(0, 0)),
             (datetime(1983, 3, 29, 15, 45, 0, 0, tz.gettz('UTC')), Timestamp(417800721, 0)),
             (datetime(2017, 12, 5, 16, 33, 12, 196, tz.gettz('UTC')), Timestamp(1512491629, 196000)),
+            (datetime(2017, 12, 5, 16, 33, 12, 999999, tz.gettz('UTC')), Timestamp(1512491629, 999999999)),
```

Comment:
Wouldn't this be a truer representation of that timestamp? Then the implementation rounds and adjusts the seconds if >999999?
```suggestion
            (datetime(2017, 12, 5, 16, 33, 13, 0, tz.gettz('UTC')), Timestamp(1512491629, 999999999)),
```

Reply:
Yes, you're right. I'll change that.